tests/security/alerts/test_models.py

```
@@ -0,0 +1,52 @@
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+"""Unit tests for ``security.alerts.models``."""
+
+from security.alerts.models import AlertMetadata
+
+
+# =====================================================================
+# AlertMetadata – None-safe __post_init__
```

Ruff (0.15.10)

[warning] Comment contains ambiguous `–` (EN DASH). Did you mean `-` (HYPHEN-MINUS)?

(RUF003)